ines-test-base.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+from sklearn.base import BaseEstimator
+from sklearn.naive_bayes import GaussianNB
+from sklearn.preprocessing import StandardScaler
+from sklearn.linear_model import LogisticRegression
+from sklearn.svm import SVC
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.cluster import KMeans
+from packaging.version import parse as parse_version
+import sklearn
+
+class CustomEstimator(BaseEstimator):
+    _doc_link = "https://custom_link/{major}.{minor}/docs/{estimator_name}.html"
+    _doc_link_module = "custom_module"
+
+    def _get_url_link(self):
+        major = 2
+        minor = 19
+        estimator_name = self.__class__.__name__
+        full_url = self._doc_link.format(**locals())
+        return full_url
```

Comment:
We should change the logic in `_get_url_link()` than, because now it is supposed that _get_url_link(), _doc_link and _doc_link_module should be overwritten.